webpack.config.js

```diff
@@ -0,0 +1,27 @@
+// this file is run with npm run build
+// it uses babel to compile everything in client/src into bundle.js
+const path = require('path');
+
+module.exports = {
+  entry: path.resolve(__dirname, './client/src'),
+  output: {
+    path: path.resolve(__dirname, './client/dist'),
+    filename: 'bundle.js',
```

Comment:
we'll need to have different bundle names so we won't have conflicts. maybe add your initials

Reply:
Good point.  I made it searchBarBundle.js - easier to keep the service names for if anyone's looking at them.  Also port #s should be different for each - I used 3003.